spec/factories/item.rb

```diff
@@ -3,5 +3,13 @@
     title "item name"
     description "yummy"
     price        5
+
+
+    factory :item_with_categories do
+       after(:create) do |item, evaluator|
```

Comment:
Unused block argument - `evaluator`. If it's necessary, use `_` or `_evaluator` as an argument name to indicate that it won't be used.<br>Use 2 (not 3) spaces for indentation.